packages/drivers/routerlicious-driver/src/insecureRouterliciousTokenProvider.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
this is worry some. this is the real frs client. i don't know if we should be adding insecure version to packages designed to be used in production. i'd also prefer pr's not mix experimental, and non-experimental code.

Reply:
@tanviraumi

Reply:
ah true...i'll take this out and if needed we can add it somewhere experimental